examples/ui-demo/src/components/configuration/Authentication.tsx

```diff
@@ -14,19 +14,28 @@ import { WalletIcon } from "../icons/wallet";
 import ExternalLink from "../shared/ExternalLink";
 import { Switch } from "../ui/switch";
 import { links } from "@/utils/links";
+import { EmailModeSwitch } from "../shared/EmailModeSwitch";
 
 export const Authentication = ({ className }: { className?: string }) => {
   const { auth, setAuth } = useConfigStore(({ auth, setAuth }) => ({
     auth,
     setAuth,
   }));
-  const setEmailAuth = (active: boolean) => {
+  const setEmailActive = (active: boolean) => {
     setAuth({ showEmail: active });
     Metrics.trackEvent({
       name: "authentication_toggled",
       data: { auth_type: "email", enabled: active },
     });
   };
+  const toggleEmailMode = () => {
```

Comment:
i believe we discussed before that for new users, picking one or the other will be enforced. does this toggle working in the demo app rely on our demo app just being grandfathered into this not being enforced server-side?